lib/xcode.js

```diff
@@ -170,4 +180,8 @@ async function getMaxIOSSDK () {
   return cache.sdkVersion;
 }
 
-export {getPath, getVersion, getAutomationTraceTemplatePath, getMaxIOSSDK};
+async function getMaxIOSSDK (retries = DEFAULT_NUMBER_OF_RETRIES) {
+  return retry(retries, getMaxIOSSDKWithoutRetry);
+}
+
+export {getPath, getVersion, getAutomationTraceTemplatePath, getMaxIOSSDK, getAutomationTraceTemplatePathWithoutRetry, getMaxIOSSDKWithoutRetry};
```

Comment:
last comment, please 1. add spaces around export braces, and 2. line up second line with beginning of first export column